doxygen/aliases

```diff
@@ -3,18 +3,25 @@ ALIASES += THG="The HDF Group"
 ################################################################################
 # Default URLs             (Note that md files do not use any aliases)
 ################################################################################
-ALIASES += HDFURL="docs.hdfgroup.org/hdf5"
-ALIASES += ARCURL="docs.hdfgroup.org/archive/support/HDF5/doc"
-ALIASES += RFCURL="docs.hdfgroup.org/hdf5/rfc"
-ALIASES += DSPURL="portal.hdfgroup.org/display/HDF5"
-ALIASES += DOCURL="portal.hdfgroup.org/documentation/hdf5-docs"
-ALIASES += DWNURL="portal.hdfgroup.org/downloads"
-ALIASES += AEXURL="support.hdfgroup.org/ftp/HDF5/examples"
+# Default URL for HDF Group Files
+ALIASES += HDFURL="support.hdfgroup.org"
+# URL for archived files
+ALIASES += ARCURL="\HDFURL/archive/support/ftp/HDF5"
```

Comment:
This needs to be reviewed and maybe changed to something that makes more sense.

Reply:
I know that "archive" shouldn't be used.